spec/controllers/characters_controller_spec.rb

```diff
@@ -594,16 +594,6 @@
       expect(character.galleries).to eq([])
     end
 
-    it "creates new templates when specified" do
```

Comment:
What's up with this?

Reply:
It's a duplicate of another test. I can find the test again, just a sec

Reply:
https://github.com/Marri/glowfic/blob/05d3a49010d82a6183c00eb58d253c5104491cdf/spec/controllers/characters_controller_spec.rb#L505 and https://github.com/Marri/glowfic/blob/05d3a49010d82a6183c00eb58d253c5104491cdf/spec/controllers/characters_controller_spec.rb#L597

Reply:
Ah, cool!